src/main/java/org/spongepowered/common/mixin/core/data/MixinDataHolder.java

```diff
@@ -413,9 +413,4 @@ public DataContainer toContainer() {
         return Collections.emptyList();
     }
 
-    @Override
-    public DataHolder copy() {
```

Comment:
What are you talking about "down the hierarchy", these bridge methods conflict **in the same class**. The *correct* fix was already applied on `bleeding`, by me. Why are you using a different approach?